noir-projects/noir-contracts/contracts/app_subscription_contract/src/dapp_payload.nr

```diff
@@ -4,9 +4,9 @@ use dep::aztec::protocol_types::{constants::GENERATOR_INDEX__SIGNATURE_PAYLOAD,
 use dep::authwit::entrypoint::function_call::{FunctionCall, FUNCTION_CALL_SIZE_IN_BYTES};
 
 // FUNCTION_CALL_SIZE * DAPP_MAX_CALLS + 1
-global DAPP_PAYLOAD_SIZE: u64 = 5;
+global DAPP_PAYLOAD_SIZE: u64 = 6;
 // FUNCTION_CALL_SIZE_IN_BYTES * DAPP_MAX_CALLS + 32
-global DAPP_PAYLOAD_SIZE_IN_BYTES: u64 = 129;
+global DAPP_PAYLOAD_SIZE_IN_BYTES: u64 = 130;
 
 global DAPP_MAX_CALLS: u64 = 1;
```

Comment:
My idiot brain started looking for the `DAPP_MAX_CALLS` in other files when I read the comments above, because I had not seen the value yet 😬 could we move it up hehe.

Reply:
Moved!